src/Config/CS_SDK.props

```diff
@@ -19,10 +20,9 @@
     <TestProjectDefaults Condition="$(MSBuildProjectDirectory.Contains('test'))">true</TestProjectDefaults>
     <TestProjectDefaults Condition=" '$(TestProjectDefaults)' == '' ">false</TestProjectDefaults>
     <NoWarn>MSB3539</NoWarn><!--The value of the property "BaseIntermediateOutputPath" was modified after it was used by MSBuild-->
-     <!--This is used to set the framework version for icon resource dll generation (lookup of system.drawing.dll)-->
-    <ResourceGeneration_FrameworkVersion>v4.8</ResourceGeneration_FrameworkVersion>
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
     <DebugSymbols>true</DebugSymbols>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
```

Comment:
I turned this off because it is causing issues (could not find assembly references in the app directory) with the existing `Copy local`=`false` settings in Dynamo.  We do not need the deps files at the moment

```
.deps.json - A list of dependencies, as well as compilation context data and compilation dependencies. 
Not technically required, but required to use the servicing or package cache/shared package install features. 
It can be used to configure dynamic linking to assemblies that come from packages. 
As mentioned above, .NET Core can be configured to dynamically load assemblies from multiple locations. These locations include:

App base directory (in the same folder as the entry point application, no config required)
Package cache folders (NuGet restore cache or NuGet fallback folders)
An optimized package cache or runtime packages store
The servicing index (rarely used. For Windows Update purposes.)
Shared framework (configured via runtimeconfig.json)
```